terraform-registry-manifest.json

```
@@ -0,0 +1,6 @@
+{
+    "version": 1,
+    "metadata": {
+        "protocol_versions": ["5.0"]
```

We are still compatible with the protocol 4, I think it is 4.0 or 4.2. It should be in the history in TeamCity but I don't have access to it anymore.

---

Thanks for the heads up. I was able to fetch via the releases headers:

```console
$ curl -I --silent https://releases.hashicorp.com/terraform-provider-consul/2.15.0/terraform-provider-consul_2.15.0_SHA256SUMS | grep x-terraform
x-terraform-protocol-version: 4
x-terraform-protocol-versions: 4.0, 5.0
```

```suggestion
        "protocol_versions": ["4.0", "5.0"]
```